src/main/java/uk/gov/hmcts/reform/draftstore/config/DraftStoreConfig.java

```diff
@@ -90,6 +93,6 @@ public ObjectMapper objectMapper() {
         return new ObjectMapper()
             .registerModule(new JavaTimeModule())
             .disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS)
-            .setDateFormat(new ISO8601DateFormat());
+            .setDateFormat(new StdDateFormat());
```

Comment:
what's the difference? was the old one deprecated?

Reply:
yup the its deprecated to be replaced with StdDateFormat.

Reply:
hmmm ... my appetite for risk has suddenly disappeared and Ill revert this.